packages/mdc-textfield/helper-text/README.md

```diff
@@ -0,0 +1,133 @@
+<!--docs:
+title: "Text Field Helper Text"
+layout: detail
+section: components
+excerpt: "The helper text provides supplemental information and/or validation messages to users"
+iconId: text_field
+path: /catalog/input-controls/text-fields/helper-text/
+-->
+
+# Text Field Helper Text
+
+The helper text provides supplemental information and/or validation messages to users. It appears on input field focus and disappears on input field blur by default, or it can be persistent.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/text-fields.html#text-fields-layout">Material Design guidelines: Text Fields Layout</a>
+  </li>
+</ul>
+
+
+## Usage
+
+### Using helper text
+
+MDC Text Fields can include helper text that is useful for providing supplemental
+information to users, as well for validation messages (covered below).
```

Comment:
nit: the part of this sentence that talks about validation messages...just move it within the README to be closer to where you actually talk about validation messages. Just adding (covered below) doesn't help users figure out what you are talking about.